activestorage/test/dummy/config/application.rb

```diff
@@ -19,8 +19,10 @@
 
 module Dummy
   class Application < Rails::Application
-    config.load_defaults 5.1
+    config.load_defaults 5.2
 
     config.active_storage.service = :local
+
+    config.action_controller.default_protect_from_forgery = false
```

Comment:
One action in particular [needn’t be protected from forgery](https://github.com/rails/rails/pull/30290), but Active Storage should otherwise work with forgery protection enabled. We should identify the cause of the test failures when it’s enabled before disabling it.